kotlin/workflow-testing/src/main/java/com/squareup/workflow/testing/RenderTester.kt

```diff
@@ -244,9 +245,36 @@ interface RenderTester<PropsT, StateT, OutputT : Any, RenderingT> {
    * @return A [RenderTestResult] that can be used to verify the [WorkflowAction] that was used to
    * handle a workflow or worker output or a rendering event.
    */
-  fun render(block: (RenderingT) -> Unit): RenderTestResult<StateT, OutputT>
+  fun render(block: (rendering: RenderingT) -> Unit = {}): RenderTestResult<StateT, OutputT>
```

Comment:
I completely forgot that you could name function arguments. Should be doing this all over the place.